tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -233,7 +233,7 @@ public function testGetTestContribution() {
    */
   public function testGetContributionReturnFunctionality() {
     $params = $this->_params;
-    $params['check_number'] = 'bouncer';
+    $params['contribution_check_number'] = 'bouncer';
```

Comment:
ideally both  should work - maybe  test both?

Reply:
Done @eileenmcnaughton